File: Totalizer/Models/Bet.cs
using System;

namespace Models
{
    public class Bet
    {
        public int Id { get; set; }        
        public Person Person { get; set; }
        public Event Event { get; set; }
        public int Coefficient { get; set; }
        public DateTime Date { get; set; }
        public decimal Amount { get; set; }
        public Team Team { get; set; }
    }
}

File: totalizer_UI/totalizer/src/app/bets/Models/Bet.ts
import { Person } from 'src/app/persons/Models/Person';
import {Event} from 'src/app/events/Models/Event';
import { Team } from 'src/app/teams/Models/Team';

export class Bet {
  Id: number;
  Person: Person;
  Event: Event;
  Coefficient: number;
  Date: Date;
  Amount: number;
  Team: Team;
}
